opencog/moses/moses/types.cc

```diff
@@ -202,6 +247,33 @@ std::ostream& ostream_scored_combo_tree(std::ostream& out,
 	return out << std::endl;
 }
 
+std::ostream& ostream_scored_atomese(std::ostream& out,
+									 const scored_atomese& sa,
```

Comment:
Minor comment about indentation. Ideally, you should use smart tabs (also called mixed tabs) that uses tabs for indentation and space of alignment. Hopefully your IDE supports it.

Or if you use `astyle` try to have it properly support smart/mixed tabs as well.